ts-fold-parsers.el

```diff
@@ -501,6 +508,18 @@
      . (lambda (node offset)
          (ts-fold-range-line-comment node offset "#")))))
 
+(defun ts-fold-parsers-matlab ()
+  "Rule set for MATLAB."
+  '((expression_list     . ts-fold-range-seq)
+    (function_definition . ts-fold-range-matlab-function)
+    (class_definition	 . ts-fold-range-matlab-function)
+    (if_statement	 . ts-fold-range-matlab-statements)
+    (for_statement	 . ts-fold-range-matlab-statements)
+    (while_statement	 . ts-fold-range-matlab-statements)
+    (switch_statement	 . ts-fold-range-matlab-statements)
+    (try_statement	 . ts-fold-range-matlab-statements)
+    (comment		 . ts-fold-range-matlab-blocks)))
```

Comment:
I've moved this function to the top (that's why there is the warning `Warning: function `ts-fold-parsers-matlab' defined multiple times in this file`). Make sure these functions are in alphabetic order!

Other than that, there are tabs in this function! 🤔